### PR TITLE
Ap 883 last few attributes

### PR DESCRIPTION
**Pull request by willc-work**
## What

[Link to story](https://dsdmoj.atlassian.net/browse/AP-883)

Add tests for various hard coded attributes
add method for the prospects of success value 

## Checklist

Before you ask people to review this PR:

- [x] Tests and rubocop should be passing: `bundle exec rake`
- [x] Github should not be reporting conflicts; you should have recently run `git rebase master`.
- [x] There should be no unnecessary whitespace changes. These make diffs harder to read and conflicts more likely.
- [x] The PR description should say what you changed and why, with a link to the JIRA story.
- [x] You should have looked at the diff against master and ensured that nothing unexpected is included in your changes.
- [x] You should have checked that the commit messages say why the change was made.

### CODE REVIEW

File: app/services/ccms/attribute_value_generator.rb [hunk 1]
```diff
@@ -179,6 +179,24 @@ def lead_proceeding_category_of_law_code(_options)
...
+        'Marginal'
+      when 'poor'
+        'Poor'
+      when 'borderline'
+        'Borderline'
+      when 'uncertain'
+        'Uncertain'
+      else
+        "Could not find match for #{check_value}"
+      end
```
Comment (reggieb) [inline app/services/ccms/attribute_value_generator.rb]: A `case` statement is a little over-kill when each of the options simply return a small string. I'd suggest:

```ruby
PROSPECTS_OF_SUCCESS = {
  likely: 'Good',
  marginal: 'Marginal',
  poor: 'Poor',
  borderline: 'Borderline'
  uncertain: 'Uncertain'
}.freeze

attr_reader :legal_aid_application
delegate :merits_assessment, to: :legal_aid_application

def ccms_equivalent_prospects_of_success(_options)
  PROSPECTS_OF_SUCCESS[merits_assessment.success_prospect.to_sym] || "Could not find match for #{check_value}"
end
```

File: spec/services/ccms/case_add_requestor_spec.rb [hunk 2]
```diff
@@ -210,6 +210,11 @@ module CCMS # rubocop:disable Metrics/ModuleLength
                capital_contribution: 0.0
       end
 
+      let(:merits_assessment) do
+        double 'MeritsAssesment',
+               success_prospect: 'Good'
```
Comment (reggieb) [inline spec/services/ccms/case_add_requestor_spec.rb]: Why are you using a double, when there is a perfectly good real object to use? Also if this does represent a `MeritsAssessment` there is no "Good" `success_prospect`. I think the value should be "likely". So:
```ruby
let(:merits_assessment) { create :merits_assessment, success_prospect: 'likely' }
```

File: spec/services/ccms/case_add_requestor_xml_blocks_spec.rb [hunk 3]
```diff
@@ -75,6 +77,32 @@ module CCMS # rubocop:disable Metrics/ModuleLength
         end
       end
 
+      context 'FAMILY_PROSPECTS_OF_SUCCESS' do
+        it 'returns the ccms equivalent prospect of success for likely' do
+          allow(legal_aid_application.merits_assessment).to receive(:success_prospect).and_return('likely')
```
Comment (reggieb) [inline spec/services/ccms/case_add_requestor_xml_blocks_spec.rb]: So a mock is being stubbed to return a value in a spec .... so what, if anything, is being tested here?

Comment (willc-work) [reply spec/services/ccms/case_add_requestor_xml_blocks_spec.rb]: I was aiming to test that the prospects of success assigned by the Apply service were translated to the correct output, e.g. when Apply assigns 'likely' this is translated to the CCMS equivalent and populates the block with 'Good'

Comment (stephenrichards) [reply spec/services/ccms/case_add_requestor_xml_blocks_spec.rb]: that seems reasonable to me, or am I missing something?

**Pull request merged**